src/Runtime/Core/Maths/SpecialFunctions.cs

```diff
@@ -3247,11 +3352,19 @@ public static double LogisticLn(double x)
         public static double Log1Plus(double x)
         {
             Assert.IsTrue(Double.IsNaN(x) || x >= -1);
-            if (x > -1e-3 && x < 6e-2)
+            if (x > -1e-3 && x < 2e-3)
```

Comment:
You shouldn't change this.

Reply:
Here I'm changing these numbers to values they had before https://github.com/dotnet/infer/pull/201/commits/7d9d0b6e1c3ada4a2f5da71e3f56652d10ce7090
In that commit explicitly truncated series were replaced with series summated until result stabilized, which allowed to increase the interval of applicability.
Now that we're going back to inlined explicitly truncated series we either need to make intervals where we use them as they were before the mentioned commit, or use longer series. E.g. `log(1 + 6e-2)` needs a 12th degree polynomial. I went with the first option, if you prefer I can change it to the 2nd.

Reply:
I prefer the 2nd.  If runtime is an issue then we can subdivide the range further and save the 12th degree polynomial for the cases where it is needed.

Reply:
OK, using longer series. For `log(1-exp(x))`, where `x < -3.5` 10th degree was enough.